docs/source/notebooks/AR.ipynb

```diff
@@ -16,7 +16,7 @@
   },
```

Comment:
<p>Maybe you can explain what "the variance of the innovation term" mean for the model, and why you pass the data as <code>y - center</code> ?</p><p>As an aside, the Normal(0, 1) on <code>theta_</code> doesn't assume <code>-1 &lt; k_ &lt; 1</code> -- it assumes 95% of the probability mass will be between -2 and 2 (2 stds). Normal(0, 0.5) would assume [-1, 1].</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3899/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>